stable/yugabyte/templates/certificates.yaml

```diff
@@ -0,0 +1,123 @@
+{{- $root := . -}}
+---
+{{- if $root.Values.tls.certManager.enabled }}
+{{- if not $root.Values.tls.certManager.useClusterIssuer }}
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name:  {{ (printf "%s-bootstrap" (include "yugabyte.tls_cm_issuer" $root)) | quote }}
+  namespace: "{{ $root.Release.Namespace }}"
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ $root.Values.oldNamingStyle | ternary "yugabyte-ca" (printf "%s-ca" (include "yugabyte.fullname" $root)) }}
+  namespace: "{{ $root.Release.Namespace }}"
+spec:
+  isCA: true
+  privateKey:
```

Comment:
This may be a naive question but I assume that once created as part of the initial `helm install`, this Certificate is not updated by cert-manager on `helm upgrade`s because none of its parameters have changed?

Reply:
cert-manager has a couple of restrictions on how it will update the secrets containing the certificates, whenever the Certificate resource is updated. But as long as it doesn't detect any changes or reasons to re-issue a certificate the actual secrets will remain untouched.